Makefile

```diff
@@ -63,6 +63,9 @@ test:
 	./bin/local-startup.sh;
 	go test -v -cover -p 1 ./... -count=1
 
+test-integration:
+	go test -v -cover -tags=integration ./internal/indexer/db/...
```

Comment:
I wanted to restrict this command to only integration tests for now. I'll find how we can run only integration tests with this command (excluding unit-tests)